test/fixtures/results/operation-example-results.ts

```diff
@@ -27,9 +30,43 @@ export const operationExampleResultFailuresWarningsString = `  GET:/harryPotter
     - Warning: This object is missing non-required properties that were unable to be validated, including middleName. Path: body. Found 2 times
 `;
 
+export const operationExampleResultFailuresWarningsStructure = (function () {
+  const result: ApiResults = {
+    apiSummary: {
+      totalPass: 0,
+      totalWarn: 1,
+      totalFail: 1,
+      totalRun: 1,
+      runDateTime: mockedSystemTime,
+    },
+  };
+  result[operationExampleResultFailuresWarnings.operationId] = {
+    endpointSummary: {
+      totalPass: 0,
+      totalWarn: 1,
+      totalFail: 1,
+      totalRun: 1,
+    },
+  };
+  result[operationExampleResultFailuresWarnings.operationId][
+    operationExampleResultFailuresWarnings.exampleGroupName
+  ] = {
+    errors: [...responseFailuresMap].map(([, value]) => ({
+      message: value.message,
+      count: value.count,
+    })),
+    warnings: [...responseOneWarningMap].map(([, value]) => ({
+      message: value.message,
+      count: value.count,
+    })),
+  };
+  return result;
+})();
```

Comment:
To me it's a bit confusing that these are defined as functions. If we want to avoid this we could get the `const result` all set up and then `export const operationExampleResultFailuresWarningsStructure = {...result}`

This notation is fine with me too though if you like it better, not a strong preference for the other

Reply:
Agreed.  I did this because defining then modifying an exported constant seemed odd, so I wrapped it in a function to make it all one operation.  On further thought I think I like using a separate export statement after the set up instead, see new commit.